rust/integration-tests/expected_db_output_files/imported_mainnet_txns/events_processor/308783012/events.json

```diff
@@ -10,8 +10,9 @@
       "index": "3723914",
       "token": "0x6fd8ee875fb55e03b1f9643d3f229f0b077a71cfe7ffc8ba7e09ff4392950e5c"
     },
-    "indexed_type": "0x4::collection::MintEvent",
-    "event_index": 0
+    "inserted_at": "2024-10-15T04:55:32.404918",
```

Comment:
We should strip this field in the `db_values`

Reply:
yes, somehow it got removed.. will add back in